bundle/language_en.properties

```diff
@@ -2001,12 +2011,12 @@ angal.ward.telephone
 angal.ward.telephoneedit                                                         = Telephone
 angal.ward.wardwithpharmacy                                                      = Ward with pharmacy
 angal.wardpharmacynew.selectward                                                 = Select a ward
-angal.wardpharmacynew.ward                                                       = Another ward
+angal.wardpharmacynew.ward                                                       = Another ward'
+angal.wardpharmacy.stockwardledger.txt                                           = StockWard Ledger
 angal.xmpp.communication                                                         = Communication
 angal.xmpp.contacts                                                              = Contacts
 angal.xmpp.getinfo                                                               = Get info
-angal.xmpp.info                                                                  = info
-angal.xmpp.isnowoffline                                                          = is now offline
+angal.xmpp.info                                                                  = infoangal.xmpp.isnowoffline                                                          = is now offline
```

Comment:
I think there's an error on new lines here

Reply:
Thanks; too many project; working too fast, and not enough sleep :(